### ISSUE TITLE
Circular dependency on the state vector generation and prior emission compilation

### ISSUE THREAD

**Issue by laestrada** [geoschem/integrated_methane_inversion#253]
### Name and Institution (Required)

Name: Lucas Estrada
Institution: Harvard ACMG

### Description of your issue or question
A circular dependency has been created in the IMI. The state vector file generation requires that the prior emissions hemco simulation has been run, but the template directory and the prior hemco emissions simulation require the StateVector.nc file to define the simulation bounds to generate the emissions.

@msulprizio can we revert the recent change and download a pre-archived global emissions file? Or do you see a better solution?

**Comment by msulprizio** [comment]
After further discussion, Lucas and I decided the solution would be to restore the use of archived HEMCO standalone output in the generation of the state vector. I have rerun the HEMCO standalone simulations for 2023 on 0.25x0.3125, 0.5x0.625, 2.2.5, and 4x5 degree global grids for 2023 saving out the annual mean CH4 emissions. This should sufficiently capture grid boxes with emissions using the latest CH4 emissions available (from GEOS-Chem 14.4.0 configuration).

Fixes have been implemented in #255.
